tests/em/static/test_DC_miniaturize.py

```diff
@@ -1,5 +1,5 @@
 from SimPEG.electromagnetics.static import resistivity as dc
-from SimPEG.electromagnetics.static.utils.static_utils import gen_DCIPsurvey
+from SimPEG.electromagnetics.static.utils.static_utils import generate_dcip_sources_line
```

Comment:
again, leave these as the old way so you can test the deprecation and the new version (and test that it reproduces the functionality)

Reply:
I'm not sure this one is totally necessary. It's testing things once a survey has been created. Creating the survey itself isn't really part of this test.

Reply:
There are tests that call gen_DCIPsurvey still. I don't see this as necessary